deploy/infrastructure/modules/terraform-aws-dss/TFVARS.md

```diff
@@ -0,0 +1,226 @@
+<!-- 
+# This file has been automatically generated by /deploy/infrastructure/utils/generate_terraform_variables.py.
+# Please do not modify manually.
+ -->
+
+# Terraform variables
+
+The following sections describe the variables of this terraform module.
+
+## terraform-aws-dss
+
+### aws_region
+
+*Type: `string`*
+
+AWS region
+List of available regions: https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/using-regions-availability-zones.html#concepts-regions
+Currently, the terraform module uses the two first availability zones of the region.
+
+Example: `eu-west-1`
+
+
+### aws_instance_type
+
+*Type: `string`*
+
+AWS EC2 instance type used for the Kubernetes node pool.
+
+Example: `m6g.xlarge` for production and `t3.medium` for development
+
+
+### aws_route53_zone_id
+
+*Type: `string`*
+
+AWS Route 53 Zone ID
+This module can automatically create DNS records in a Route 53 Zone.
+Leave empty to disable record creation.
+
+Example: `Z0123456789ABCDEFGHIJ`
+
+
+### app_hostname
+
+*Type: `string`*
+
+Fully-qualified domain name of your HTTPS Gateway ingress endpoint.
+
+Example: `dss.example.com`
+
+
+### crdb_hostname_suffix
+
+*Type: `string`*
+
+The domain name suffix shared by all of your CockroachDB nodes.
+For instance, if your CRDB nodes were addressable at 0.db.example.com,
+1.db.example.com and 2.db.example.com, then the value would be db.example.com.
+
+Example: db.example.com
+
+
+### cluster_name
+
+*Type: `string`*
+
+Name of the kubernetes cluster that will host this DSS instance (should generally describe the DSS instance being hosted)
+
+Example: `dss-che-1`
+
+
+### node_count
+
+*Type: `number`*
+
+Number of Kubernetes nodes which should correspond to the desired CockroachDB nodes.
+**Always 3.**
+
+Example: `3`
+
+
+### aws_kubernetes_storage_class
+
+*Type: `string`*
+
+AWS Elastic Kubernetes Service Storage Class to use for CockroachDB and Prometheus persistent volumes.
+See https://docs.aws.amazon.com/eks/latest/userguide/storage-classes.html for more details and
+available options.
+
+Example: `gp2`
+
+
+### image
+
+*Type: `string`*
+
+Full name of the docker image built in the section above. build.sh prints this name as
+the last thing it does when run with DOCKER_URL set. It should look something like
+gcr.io/your-project-id/dss:2020-07-01-46cae72cf if you built the image yourself as
```

Comment:
nit: Since this is AWS-specific, should this example use whatever the AWS container registry is?  (gcr.io is Google Cloud's container registry)

Not worth a research project though, so if that's hard to figure out, then as-is is fine.

Reply:
Good catch. AWS has its own containers repository (Elastic Container Registry).
I updated the documentation in build.sh and made a reference in TFVARS.md files.